### PR TITLE
Create test for covering the "Php" and "Php-debug" plugins

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Create test for covering the "Php" and "Php-debug" plugins

### Screenshot/screencast of this PR

Issue: https://github.com/eclipse/che/issues/19503
[php-test.log](https://github.com/eclipse/che/files/6265164/php-test.log)

### What issues does this PR fix or reference?

Issue: https://github.com/eclipse/che/issues/19503

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/e2e/tests/plugins/PhpPlugin.spec.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            await browserTabsUtil.navigateTo(factoryUrl);
+        });
+
+        test('Wait until created workspace is started', async () => {
+            await ide.waitAndSwitchToIdeFrame();
+            await ide.waitIde(TimeoutConstants.TS_SELENIUM_START_WORKSPACE_TIMEOUT);
+            await projectTree.openProjectTreeContainer();
+            await projectTree.waitProjectImported(projectName, subRootFolder);
+
+            workspaceName = await WorkspaceNameHandler.getNameFromUrl();
```
Comment (Katka92) [inline tests/e2e/tests/plugins/PhpPlugin.spec.ts]: What about moving this directly to the `Delete workspace` test - so the variable initialization and usage would be at one place and there would be no need to make the `workspaceName` variable global for this class. 
E.g. as you did it in another test: https://github.com/eclipse/che/pull/19497/files#diff-6e013c167f0c38d47f74ef27f9a9145ed154e634ae10f5b93c7befca735b6e6aR75

**Review by Katka92 — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
[crw-ci-test --rebuild]

**Comment by Ohrimenko1988** [comment]
[crw-ci-test --rebuild]

**Comment by Ohrimenko1988** [comment]
[crw-ci-test --rebuild]

**Pull request merged**